docs/setting-up/client/mongodb.md

```diff
@@ -245,6 +245,46 @@ pmm-admin remove mongodb SERVICE_NAME
 
 - `SERVICE_NAME`: The name the service was added as. (Find it with `pmm-admin list`.)
 
+## Advanced Options
```

Comment:
## Advanced Options:

Add this sentence as we should not have orphan headings.

"The advanced options while setting up PMM to monitor MongoDB database instance are as follows."